action/action.py

```diff
@@ -124,6 +124,10 @@ async def preflight():
                     # For push events
                     ref = event_data["ref"]
 
+                    # For tag events
+                    if ref.startswith("refs/tags/"):
+                        ref = ref.split("/")[-1]
```

Comment:
Ideally we would only set `ref` once, now we do it 2 times

Reply:
You are right. I was about to change the ref assignation with walrus operator in the "if", but it made the code less readable in my opinion.

Another improvement should be to handle better how we get the tag name. Now, if the tag name contains a /, this code will not work properly. I can create another PR if you want to fix this. But for now, this will work for most of the cases.

Reply:
Honestly, that's a) a bad tag name, and b) I truly expect that you are the only one that are using tag_events with this action

Reply:
a) I agree
b) even if I am not, people will not encounter this problem now :)